examples/Steel_Hall/Program.cs

```diff
@@ -0,0 +1,1291 @@
+                            load_is_over_total_length = true,
+                            load_is_over_total_lengthSpecified = true,
+                        };
+                        member_loads_LC4.Add(member_load_id, newMemberLoad);
+                        member_load_id++;
+                    }
+                }
+
+                try
+                {
+                    model.begin_modification("Set loads");
+                    foreach (KeyValuePair<int, member_load> memberload in member_loads_LC2)
+                    {
+                        model.set_member_load(liveLoad.no, memberload.Value);
+                    }
+                    foreach (KeyValuePair<int, member_load> memberload in member_loads_LC3)
+                    {
+                        model.set_member_load(windX.no, memberload.Value);
+                    }
+                    foreach (KeyValuePair<int, member_load> memberload in member_loads_LC4)
+                    {
+                        model.set_member_load(windY.no, memberload.Value);
+                    }
+                    foreach (KeyValuePair<int, member_load> memberload in member_loads_LC5)
+                    {
+                        model.set_member_load(snow.no, memberload.Value);
+                    }
+                }
+                catch (Exception exception2)
+                {
+                    model.cancel_modification();
+                    logger.Error(exception2, "Something happened while load transfer" + exception2.Message);
+                    throw;
+                }
+                finally
+                {
+                    try
+                    {
+                        model.finish_modification();
+                    }
+                    catch (Exception exception)
+                    {
+                        logger.Error(exception, "Something went wrong while finishing load transfer\n" + exception.Message + "\n");
+                        model.reset();
+                    }
+                }
+#if RFEM
+                #region generate mesh and get mesh statistics
+                calculation_message[] meshGenerationMessage = model.generate_mesh(true);
+                if (meshGenerationMessage.Length != 0)
+                {
+                }
+                mesh_statistics_type mesh_Statistics = model.get_mesh_statistics();
+                Console.WriteLine("Number of mesh nodes: " + mesh_Statistics.node_elements);
+                Console.WriteLine("Number of 1D elements: " + mesh_Statistics.member_1D_finite_elements);
+                Console.WriteLine("Number of surface element: " + mesh_Statistics.surface_2D_finite_elements);
+                Console.WriteLine("Number of volume elements: " + mesh_Statistics.solid_3D_finite_elements);
+                #endregion
+#endif
+                calculation_message[] calculationMessages = model.calculate_all(true);
+                if (calculationMessages.Length != 0)
+                {
```

Comment:
You can add here printing of calculation message if something is unsucessful during calculation